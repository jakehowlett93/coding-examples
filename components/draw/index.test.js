import Draw from './index';

describe('draw', () => {
	const state = {
		textGuess: 'hello',
		room: {
			roundType: 'draw'
		}
	};
	let returnValue;
	beforeEach(() => {
		returnValue = Draw({ state });
	});
	it('should draw the draw component', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
