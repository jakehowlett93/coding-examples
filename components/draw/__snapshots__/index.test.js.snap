// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`draw should draw the draw component 1`] = `
<div
  className="draw"
>
  <div
    className="canvasContainer"
  >
    <canvas
      className="gameCanvas"
      height="720"
      id="canvas"
      onMouseDown={[Function]}
      onMouseMove={[Function]}
      onMouseUp={[Function]}
      width="1280"
    />
  </div>
  <ColorPicker
    state={
      Object {
        "room": Object {
          "roundType": "draw",
        },
        "textGuess": "hello",
      }
    }
  />
  <ThicknessSlider
    state={
      Object {
        "room": Object {
          "roundType": "draw",
        },
        "textGuess": "hello",
      }
    }
  />
  <button
    className="submit"
    onClick={[Function]}
    type="button"
  >
    Submit
  </button>
</div>
`;
